backend/src/util/utilities.ts

```diff
@@ -14,3 +14,22 @@ export const decimalSeparatorByLanguage = (a: Decimal, language: string): string
   i18n.setLocale(rememberLocaleToRestore)
   return result
 }
+
+export const fullName = (firstName: string, lastName: string): string => {
+  return [firstName, lastName].filter(Boolean).join(' ')
+}
+
+export const userName = (firstName?: string, lastName?: string): string | null => {
+  let name: string | null
+  if (firstName && lastName) {
+    name = firstName + ' ' + lastName
+  } else if (firstName && !lastName) {
+    name = firstName
+  } else if (!firstName && lastName) {
+    name = lastName
+  } else {
+    name = null
+  }
+
+  return name
+}
```

Comment:
I tested the fullName method (in the browser console) I got the following results:
- if firstName & lastName are null or '' => result is ''
- if firstName or lastName are null or '' => result is `${filledVariable}`
- if firstName and lastName are not null or '' => result is `${firstName} ${lastName}`

So do we still need this method? I think you can delete it:
```suggestion
```

Reply:
It was a requirement from @Mogge to test for these variables before inserting them in the username column of the transactions table in database.
I can't follow your second result, because the code contains the following rules:
- if (firstName && !lastName)     name = firstName 
- if (!firstName && lastName)     name = lastName